src/tests/provides.test.ts

```diff
@@ -1,24 +1,28 @@
-import { productsRequest } from "../utils/client";
+import { productsRequest } from '../utils/client';
 
-test("@provides", async () => {
+test('@provides', async () => {
   const resp = await productsRequest({
     query: `#graphql
       query ($id: ID!) {
         product(id: $id) {
           createdBy { email totalProductsCreated }
         }
       }`,
-    variables: { id: "apollo-federation" },
+    variables: { id: 'apollo-federation' },
   });
 
+  const totalProductsCreated: number =
```

Comment:
move this line after the first expectation ... if the response doesn't contain `data` it'll blow up with a confusing "property product does not exist on undefined" error

Reply:
(or use `?.` everywhere)

Reply:
👍